queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_in_notification.py

```diff
@@ -68,10 +68,24 @@ def _get_pdfs(
 
         # add receipt
         corp_name = business.get('legalName')
+
+        # Debugging logger for #24361, will remove later
```

Comment:
Later when? Is there a future tech debt / cleanup ticket to remove this?

I don't mind debugging / logging, but when you say you're going to remove it, I'm going to call you on it 😆

Reply:
after this PR got merged and pushed to Main, I will try to file a continuation in in DEV and monitor the console. If everything works as expected. I will create another PR before closing this ticket to remove these.

Reply:
Sounds good.